hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -277,14 +277,6 @@ public class HRegion implements HeapSize, PropagatingConfigurationObserver, Regi
   public static final String RECOVERED_EDITS_IGNORE_EOF =
     "hbase.hregion.recovered.edits.ignore.eof";
 
-  /**
-   * Whether to use {@link MetaCellComparator} even if we are not meta region. Used when creating
-   * master local region.
-   */
-  public static final String USE_META_CELL_COMPARATOR = "hbase.region.use.meta.cell.comparator";
```

Comment:
Not a huge deal, but why move this constant in this PR? Doesn't seem related?

Reply:
> Not a huge deal, but why move this constant in this PR? Doesn't seem related?

Because the new InnerStoreCellComparator which placed in hbase-common module reference it too, and the hbase-common should not depend on hbase-server.  
Thanks.